packages/core/src/document.ts

```diff
@@ -418,14 +418,10 @@ class Document extends EventEmitter {
       // If they have the same block number fall through to fallback mechanism
     }
 
-    // The anchor status is the same between both logs.  If either log has a 'nonce' that means
-    // multiple updates have been squashed into a single anchor window, so prefer the log with the
-    // higher nonce (which indicates it has had more writes).
-    const nonce1 = (state1.next?.metadata.nonce ?? state1.metadata.nonce) || 0
-    const nonce2 = (state2.next?.metadata.nonce ?? state2.metadata.nonce) || 0
-    if (nonce1 < nonce2) {
+    // The anchor states are the same for both logs. Compare log lengths and choose the one with longer length.
+    if (state1.log.length > state2.log.length) {
```

Comment:
I think this case is backwards.  We're supposed to return true when selecting state2's log, and false when selecting state1's log